### ISSUE TITLE
declutter log file

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#966]
Make a filter that looks at mb2's log file and summarizes various recurring log lines and highlights unusual occurrences.

I currently check for new crates and new npm packages in the morning and do a deploy if anything has been updated. That means I'm deploying a new version most mornings. When I do this, I scan the log lines for anything that is unusual, which means I've trained my eyes to skip over various actix messages, e.g.,
```
2022-06-02T03:29:30.628184730Z ERROR [actix_http::h1::dispatcher] stream error: Request parse error: Invalid Header provided
2022-06-02T03:36:57.416867525Z ERROR [actix_http::h1::dispatcher] stream error: Request parse error: Invalid Header provided
2022-06-02T09:44:52.765526942Z ERROR [actix_http::h1::dispatcher] handler did not read whole payload and dispatcher could not drain read buf; return 500 and close connection
```
as well as warnings about commands being out of sync:
```
2022-06-02T01:19:46.391719678Z INFO  [mb2::table] out of sync expected 97, got 33, seat: SeatIndex(2), table: 4125, hand: Some(HandId(270049))
```
Although that is what I do, it's an error-prone methodology that makes it more likely that I'll miss an extremely rare issue. I have no reason to believe that there are extremely rare issues that I'm missing, but getting rid of the chaff is the way to go. 

The `actix-http` messages are being done through the `tracing` crate, so eventually using [`tracing-actix-web`](https://docs.rs/tracing-actix-web/latest/tracing_actix_web/) makes sense, but if I'm writing a script that is already processing the out of sync messages, it probably makes sense to initially have that do the work for actix related messages.

Whatever summary I do needs to, at a minimum, provide for counts of what has been seen and to tuck the original data away for further analysis. My guess is most of the http errors we see are related to software other than mb2 connecting to the server (e.g., port scanners, web crawlers) and the out of sync messages are simply due to lag. However, if the frequency of either of those rises significantly, it could represent a bug in mb2 (possibly coming from a regression in a crate) and so we don't want to miss that sort of error just because we want other errors to stand out.

FWIW, I've known about this issue forever. I only created this issue now because I'm anticipating an increase in both use of this site as well as potentially more probing for vulnerabilities.

**Comment by ctm** [comment]
I got rid of the `out of sync` warnings yesterday. I haven't yet figured out how to cut down the `actix-http` ones. I'll do that "soon", but it's still not a high prioriity.

**Comment by ctm** [comment]
I've eliminated the `actix-http` ones. So, mb2 has gone from being overly chatty to silencing warnings that haven't mattered yet but may matter sometime. The solution, I believe, is to internally note these warnings and then summarize them. As such, I've opened two new issues: one for out of sync warnings (#1055) and the other for http warnings (#1056).

I'm not in a hurry to do either, so neither are labeled `high priority`.
